lib/fpsemi.gi

```diff
@@ -73,7 +73,7 @@ InstallMethod( \<,
     [ IsElementOfFpSemigroup, IsElementOfFpSemigroup ],
     0,
     function( x1, x2 )
-			local S, RWS;
+                        local S, RWS;
```

Comment:
maybe 4 (or 3?) wide tabs?

Reply:
I've had another run through this file, super inconsistent indentation it seems!